src/analysis_file.py

```diff
@@ -75,9 +75,9 @@ def generate(cls, user, data, pipeline_configuration, raw_data_dir,  csv_by_mess
         # Export listening group bool keys in analysis files headers only when running kakuma_pipeline because
         # dadaab does not have listening groups.
         if pipeline_configuration.pipeline_name == "kakuma_pipeline":
-            export_keys.extend("repeat_listening_group_participant")
+            export_keys.extend(["repeat_listening_group_participant"])
```

Comment:
If you're extending with a one element list, consider using .append() instead

Reply:
True, asante!